Description: Catchsegv has been removed in recent glibc 2.35.
Author: Gianfranco Costamagna <locutusofborg@debian.org>
Last-Update: 2022-02-07

--- mutter-41.3.orig/meson.build
+++ mutter-41.3/meson.build
@@ -313,7 +313,7 @@ if have_tests
     ],
     exe_wrapper: [
       default_test_wrappers,
-      find_program('catchsegv'),
+      #find_program('catchsegv'),
       find_program('xvfb-run'), '-a', '-s', '+iglx -noreset',
     ],
     timeout_multiplier: 10,
